server/test/unit/auth/test_auth.py

```diff
@@ -79,27 +82,27 @@ def test_auth_test(self):
             assert r.history[0].status_code == 302
             assert r.url == f"{server}/auth/pbmc3k.cxg/"
 
-            r = session.get(f"{server}/auth/pbmc3k.cxg/api/v0.2/config")
-            data_config = r.json()
-            assert data_config["config"]["authentication"]["is_authenticated"]
-            assert data_config["config"]["authentication"]["username"] == "test_account"
-            assert data_config["config"]["parameters"]["annotations"]
+            config = session.get(f"{server}/auth/pbmc3k.cxg/api/v0.2/config").json()
+            userinfo = session.get(f"{server}/auth/pbmc3k.cxg/api/v0.2/userinfo").json()
+            assert userinfo["userinfo"]["is_authenticated"]
+            assert userinfo["userinfo"]["username"] == "test_account"
+            assert config["config"]["parameters"]["annotations"]
 
             r = session.get(f"{server}/{logout_uri}")
             # check that the logout redirect worked
             assert r.history[0].status_code == 302
             assert r.url == f"{server}/auth/pbmc3k.cxg/"
-            r = session.get(f"{server}/auth/pbmc3k.cxg/api/v0.2/config")
-            data_config = r.json()
-            assert not data_config["config"]["authentication"]["is_authenticated"]
-            assert data_config["config"]["authentication"]["username"] is None
-            assert data_config["config"]["parameters"]["annotations"]
+            config = session.get(f"{server}/auth/pbmc3k.cxg/api/v0.2/config").json()
+            userinfo = session.get(f"{server}/auth/pbmc3k.cxg/api/v0.2/userinfo").json()
+            assert not userinfo["userinfo"]["is_authenticated"]
+            assert userinfo["userinfo"]["username"] is None
```

Comment:
could you swap the assert statements for self.assertEqual? its more readable and that seems to be the pattern in the rest of the codebase

Reply:
done